src/binfmt/binfmt.c

```diff
@@ -140,6 +151,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "help",       no_argument, NULL, 'h'            },
                 { "version",    no_argument, NULL, ARG_VERSION    },
                 { "cat-config", no_argument, NULL, ARG_CAT_CONFIG },
+                { "tldr",       no_argument, NULL, ARG_TLDR       },
```

Comment:
hmm, i wonder if we should name this `--cat-config-tldr`, since it is just a special flavour of `--cat-config` after all, and the relationship to that might be interesting to make visibale?

(not totally convinced myself, feel free to ignore)

Reply:
I thought about this, but it's quite a lot of typing… `--tldr` is about a quick peek, and it would be annoying to have to type such a long name. (Even with autocompletion it'd be quite a few keystrokes to disambiguate from `--cat-config`.)   I've grown to prefer the current name.